tools/stonith_admin.c

```diff
@@ -282,7 +282,7 @@ add_stonith_params(const gchar *option_name, const gchar *optarg, gpointer data,
 
     if (rc != 2) {
         crm_err("Invalid option: -o %s: %s", optarg, pcmk_strerror(rc));
-        g_set_error(error, G_OPTION_ERROR, rc, "Invalid option: -o %s: %s", optarg, pcmk_strerror(rc));
+        g_set_error(error, PCMK__RC_ERROR, rc, "Invalid option: -o %s: %s", optarg, pcmk_strerror(rc));
```

Comment:
This looks like a legacy rc, so we should convert it first (and use pcmk_rc_str())

Looking at this I wonder why we're logging this with crm_err() ... the only way anyone would see it is if they used -VV and then they'd see it twice, once from the GError (which I assume we're displaying somewhere) and once as an error log

Reply:
Fixed, and removed the crm_err call in add_env_params that seems likewise useless.